docs/examples/basic_retrieval.ipynb

```diff
@@ -406,9 +406,9 @@
       "source": [
         "user_model = tf.keras.Sequential([\n",
         "  tf.keras.layers.experimental.preprocessing.StringLookup(\n",
-        "      vocabulary=unique_user_ids),\n",
-        "  # We add 2 to account for unknown and mask tokens.\n",
-        "  tf.keras.layers.Embedding(len(unique_user_ids) + 2, embedding_dimension)\n",
+        "      vocabulary=unique_user_ids, mask_token=None),\n",
+        "  # We add additional embedding to account for unknown tokens.\n",
```

Comment:
Slight wording correction: "We add _an_ additional embedding..."

Reply:
Sure! Just updated it. Many thanks!